faiss.py

```diff
@@ -0,0 +1,54 @@
+
+import pickle
+import faiss
+import abc
+
+
+
+class Indexer(abc.ABC):
+
+  @abc.abstractmethod
+  def __init__(self, dim, param, nprobe):
```

Comment:
You can remove __init__ method, these parameters are not required

Reply:
ok